doc/api/webcrypto.md

```diff
@@ -66,7 +66,7 @@ async function generateEcKey(namedCurve = 'P-521') {
 }
 ```
 
-#### ED25519/ED448/X25519/X448 Elliptic curve key pairs
+#### ED25519/ED448/X25519/X448 key pairs
```

Comment:
I'm wondering if it would make sense to elaborate here but honestly if you don't know what ES25519 is before reading these docs it probably won't help and if you did it won't help either..

Reply:
https://github.com/nodejs/node/pull/41542#discussion_r785445205